.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+## Description
+(...)
+
+Fixes #(insert issue number here if applicable)
```

Comment:
do you require people to open an issue before creating a pull request?

Reply:
I'd say no—no particular need for it, if someone is intrepid enough to fix their own bug.